mmseg/models/backbones/beit.py

```diff
@@ -0,0 +1,558 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+import warnings
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import build_norm_layer
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.cnn.bricks.transformer import FFN
+from mmcv.cnn.utils.weight_init import (constant_init, kaiming_init,
+                                        trunc_normal_)
+from mmcv.runner import BaseModule, ModuleList, _load_checkpoint
+from scipy import interpolate
```

Comment:
We may abandon `scipy` because it is not pre-installed. Could `scipy.interpolate.interp2d` be replaced by `F.interpolate` or something else?